apps/mail/components/create/editor-autocomplete.ts

```
@@ -0,0 +1,255 @@
+
+              view.dispatch(tr);
+
+              usedSuggestions.add(suggestion);
+
+              return true;
+            } catch (error) {
+              console.error('Error applying suggestion:', error);
+              return false;
+            }
+
+            return true;
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)